ha_cluster_exporter.go

```diff
@@ -57,6 +59,20 @@ func NewMetricDesc(subsystem, name, help string, variableLabels []string) *prome
 	return prometheus.NewDesc(prometheus.BuildFQName(NAMESPACE, subsystem, name), help, variableLabels, nil)
 }
 
+func CheckExecutables(paths ...string) error {
+	// check that all executables we rely on exist and are executables
```

Comment:
`nit` inn a `golint` way of thing the comment should be like
functioname and desc
\\ CheckExecutables  check that all executables we rely on exist and are executables
func foof

Reply:
I don't quite understand what you mean. Do you want the comment moved above the declaration?

Reply:
@stefanotorresi take this as example:
https://github.com/kubernetes/kubernetes/blob/master/cmd/genutils/genutils.go#L25

Reply:
for a public function in go normally the convention is:

`// funcname blalbla`
func foo